src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -322,20 +322,52 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                             continue;
 
                         case "nullable":
```

Comment:
>case "nullable": [](start = 24, length = 16)

Please update the speclet with this information about command-line. 
Also we should update the help message (filing an issue for that one seems fine if you prefer).